### PR TITLE
Adds RMRK Logo and Squid Configuration.

### PR DESCRIPTION
**Pull request by steven2308**

### CODE REVIEW

File: registry/mainnet/interchain/squid.tokenlist.json [hunk 1]
```diff
@@ -6,5 +6,29 @@
...
+      "tokenId": "0x41e276f2a123d67022502cdfa56ce754fc6eb27f13e7ac3da4c69a08ac668169",
+      "tokenAddress": "0x7e5738bDabc8ADb3670b07eA0e7D4b0B94282E4f",
+      "symbol": "RMRK",
+      "prettySymbol": "RMRK",
+      "decimals": 18,
+      "name": "Test RMRK",
+      "originAxelarChainId": "moonbeam",
+      "transferType": "interchain",
+      "iconUrls": {
+        "svg": "https://raw.githubusercontent.com/axelarnetwork/axelar-configs/images/tokens/rmrk.png"
```
Comment (canhtrinh) [inline registry/mainnet/interchain/squid.tokenlist.json:20]: can you provide in svg format

Comment (steven2308) [reply registry/mainnet/interchain/squid.tokenlist.json]: Fixed!

**Review by ecdsafu — CHANGES_REQUESTED** [review]
Looks good, please use icons which can be displayed in a circle. I can convert them if you send them to me.
[rmrk.svg.zip](https://github.com/axelarnetwork/axelar-configs/files/13855096/rmrk.svg.zip)

**Comment by steven2308** [comment]
> Looks good, please use icons which can be displayed in a circle. I can convert them if you send them to me. [rmrk.svg.zip](https://github.com/axelarnetwork/axelar-configs/files/13855096/rmrk.svg.zip)

I uploaded a new one, thanks!

**Comment by ecdsafu** [comment]
> > Looks good, please use icons which can be displayed in a circle. I can convert them if you send them to me. [rmrk.svg.zip](https://github.com/axelarnetwork/axelar-configs/files/13855096/rmrk.svg.zip)
> 
> I uploaded a new one, thanks!

Looks great! Thank you 

@canhtrinh could we also have an optional coingeckoId property on these tokens?

**Review by canhtrinh — APPROVED** [review]

**Pull request merged**